Jenkinsfile

```diff
@@ -136,6 +136,7 @@ node ('master') {
   pushDockerTag = 'not-supposed-to-be-pushed'
   packagePush = false
   specialBranch = false
+  parallelism = 0
```

Comment:
Did you test it with `0`? Is that even a valid option for `make` command? I suggest making `1` a default value

Reply:
`0`  is converted to default: 8 for Linux and 4 for Mac [here]( https://github.com/hyperledger/iroha/pull/2027/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R251) ,  you can choose `1` and build in one thread both Linux and Mac. 
`0` not valid option,  and it not be used. 
 `make: the '-j' option requires a positive integer argument`,